test_launch_ros/test/test_launch_ros/actions/test_node.py

```diff
@@ -184,7 +186,8 @@ def test_launch_node_with_parameter_descriptions(self):
         )
         self._assert_launch_no_errors([node_action])
 
-        expanded_parameter_arguments = node_action._Node__expanded_parameter_arguments
+        for node in node_action.ros_exec.nodes:
+            expanded_parameter_arguments = node.expanded_parameter_arguments
```

Comment:
@mlanting hmm, should these be extending the `expanded_parameter_arguments` list instead of overwriting? Same below.